packages/office-ui-fabric-react/src/components/Persona/Persona.styles.ts

```diff
@@ -65,14 +65,6 @@ export const getStyles = (props: IPersonaStyleProps): IPersonaStyles => {
         selectors: {
           '.contextualHost': {
             display: 'none'
-          },
-
-          ':hover': {
```

Comment:
@cliffkoh I believe the hover selector was put on the root so that the primary text changes when hovering on any part of the Persona. Moving it down makes it so it only hovers when you directly hover over the primary text.

Reply:
Actually I believe that in both cases, it would only trigger if you hover over primary text. Note that this selector wasn't directly on root, it had primaryText NESTED within root.

The net effect of this change should primarily be that the primaryText hover selector has dropped in specificity...

LMK if this does not make sense to you or if I am missing something here (totally possible!)

Reply:
One of the things we do when we convert from SASS to merge-styles is to flatten all specificity, in order to allow overwrites through the styles interface rather than resort to specificity wars. My interpretation of the style before this PR was that it was just a conversion mistake...

Reply:
Here is if I just change the color in that selector to red leaving the current styles
![2019-07-25 22 56 58](https://user-images.githubusercontent.com/29514833/61929385-fb8a9d80-af2f-11e9-982f-9fa2ad3a3ddd.gif)

And here is if I bring in your changes and change the hover color to red
![2019-07-25 22 59 15](https://user-images.githubusercontent.com/29514833/61929529-0e04d700-af30-11e9-86b6-9f2e709abd71.gif)

Looks like it’s exactly as I said. The intention was to change the primary text color when hovering on any part of the persona. You might flatten your selector in a different way:
```tsx
':hover .ms-Persona-primaryText': {
  color: palette.neutralDark
}
```

…and instead of the hard coded string className you could use the `GlobalClassNames.primaryText`

Reply:
Thanks @vitalius1, I'll look into this.

Reply:
@Vitalius1 Thanks for catching that - I missed the little nuance about the position in the nesting that `:hover` appeared :)  However, I'm pretty sure fixing it the way you propose would cause the original issue reported by the user to still apply. So we have to optimize for 1 or the other .. and it depends on how essential it is that hovering over any part of the persona highlights the primaryText.

@Markionium do you know/have an opinion here?

Reply:
cc: @betrue-final-final that might know about the hover intent here.

Reply:
+1 on changing the color when hovering anywhere over the persona (like it is today). We can probably check with Matt (who AFAIK owns the persona control from the design side) to be sure. Ill fire an email

Reply:
Thanks. If confirmed, I'll make the smaller change that Vitalie suggested but this will result in the original issue that is reported to still persist (but we'll just have to accept that as one of the limitations of merge-styles/the styles interface)

Reply:
@cliffkoh this limitation should be resolved with introduction of tokens. The user will just provide the desired color to a specific token without worrying about digging into the source code to match the specificity or some nested scenario.